libbpfgo.go

```diff
@@ -393,6 +393,34 @@ func (m *Module) InitGlobalVariable(name string, value interface{}) error {
 	return err
 }
 
+// GetMapsByName retrieves BPF maps with the specified name.
+func GetMapsByName(name string) []*BPFMapLow {
```

Comment:
I see a problem with returning ready-made BPFMapLow instead of map IDs: the caller might simply forget to close the maps' FDs and cause them to leak. I think forcing the caller to call `GetMapByID` for each map ID makes the caller's responsibility for management crystal.

`func GetMapsByName(name string) []int {`

WDYT?

Reply:
Sorry, I was not aware of the closing of maps' FDs. In this case, yes, it is a good idea to just return Ids and let the user handle the rest.